style.css

```diff
@@ -1,825 +1,810 @@
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::after,
+*::before {
+	box-sizing: inherit;
+}
+
 body {
-  margin: 0;
-  font-family: Montserrat, sans-serif;
+	min-width: 320px;
+	overflow-x: hidden;
+	font-family: Montserrat;
+	margin: 0;
 }
 
-.collection__img-mob {
-  display: none;
+h1 {
+	margin: 0;
+}
+
+.second__bag-dsk,
+.third-count,
+.head__desctop,
+.second__quantity,
+.second__other,
+.second__vector,
+.third__img-dsk {
+	display: none;
 }
 
 .header {
-  position: absolute;
-  z-index: 3;
-  width: 100%;
+	background-color: #fff;
 }
 
 .header__inner {
-  display: flex;
-  justify-content: space-between;
+	height: 68px;
+	display: flex;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/step3/pullRequest?prid=5898150)